UsageExamples/OoplesIndicatorLibraryExample.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Alpaca.Markets;
+using OoplesFinance.StockIndicators.Models;
+using static OoplesFinance.StockIndicators.Calculations;
+
+namespace UsageExamples
+{
+    public class OoplesIndicatorLibraryExample
+    {
+        const string paperApiKey = "REPLACEME";
+        const string paperApiSecret = "REPLACEME";
+        const string symbol = "AAPL";
+
+        public async Task RunExample()
+        {
+            var startDate = new DateTime(2021, 01, 01);
+            var endDate = new DateTime(2021, 12, 31);
+
+            var client = Environments.Paper.GetAlpacaDataClient(new SecretKey(paperApiKey, paperApiSecret));
+            var bars = (await client.ListHistoricalBarsAsync(new HistoricalBarsRequest(symbol, startDate, endDate, BarTimeFrame.Day)).ConfigureAwait(false)).Items;
+            var stockData = new StockData(bars.Select(x => x.Open), bars.Select(x => x.High), bars.Select(x => x.Low), bars.Select(x => x.Close), bars.Select(x => x.Volume), bars.Select(x => x.TimeUtc));
+
+            // can do typical indicators such as using typical bollinger band settings
+            var results = stockData.CalculateBollingerBands();
+
+            // we can also do completely custom indicators with very little effort such as below
+            //var results2 = stockData.CalculateBollingerBands(MovingAvgType.Ehlers3PoleSuperSmootherFilter, 50, 3);
```

Comment:
Create two additional methods for calculating/printing indicators - the user should understand that the same input data can be used several times and it should be clear that results processing depends on indicator type.

Reply:
If I'm understanding you correctly you want one method to handle the printing and one method to handle all calculations for the three different examples

Reply:
No. I just want to have 3 helper methods named `useBollingerBandsDefault`, `useBollingerBandsCustom`, and `useMovingAverageConvergenceDivergence` (or something like this). Each one will demonstrate how to use the `stockData` variable for calculating the indicator and printing its values.